src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -173,18 +172,18 @@ def test_WINNF_FT_S_PAT_1(self, config_filename):
     config = loadConfig(config_filename)
     num_tests = len(config)
     test_pass_threshold = 0.999 #percentage of passed test expected
-    num_passed_tests = 0.0
+    num_passed_tests = 1
     num_tests = len(config)
     max_fail_num = int ((1-test_pass_threshold) * num_tests)
-    num_failed_tests = 0.0
-    num_invalid_tests = 0.0
+    num_failed_tests = 1
+    num_invalid_tests = 1
     for request in  config:
         try:
            ref_response = computePropagationAntennaModel(request)
         except ValueError as e:
            logging.debug(e)
```

Comment:
I suggested in numerous previous comments that you log the configuration information to help with debugging. Please either address this comment or say why you disagree with it.